Doppler.Import.Subscribers.App/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace Doppler.Import.Subscribers.App
```

Comment:
what do you think about merge `Import` and `Subscribers` into a single word `ImportSubscribers`? (the same in the lot of other places)

Also, what do you think about renaming `App` as `Cli`?

```suggestion
namespace Doppler.ImportSubscribers.Cli
```

Reply:
good idea, I'm going to do it

Reply:
Ready to review

Reply:
I would replace `Import.Subscribers` by `ImportSubscribers` because there is not a hierarchical relation there. But I know that it is arguable.